### PR TITLE
feat: powerUp follows the player

### PR DESCRIPTION
**Pull request by PaablooCH**
Things added:
- If you don't have a powerUp stored, once you approach near one powerUp, it will follow you.
- You can deactivate the debugdraw of the powerUp.

### CODE REVIEW

File: Game/Scripts/PowerUpLogicScript.cpp [hunk 1]
```diff
@@ -32,6 +41,32 @@ void PowerUpLogicScript::Update(float deltaTime)
...
+			ownerRb->SetRotationTarget(errorRotation.Normalized());
+			if (powerUpManagerScript->GetSavedPowerUpType() != PowerUpType::NONE)
+			{
+				btRigidBody* btRb = ownerRb->GetRigidBody();
+				btRb->setLinearVelocity(btVector3(0.0f, 0.0f, 0.0f));
+				isSeeking = false;
+			}
+		}
+#ifdef DEBUG
+		if (powerUpManagerScript->GetDebugDraw())
```
Comment (markmeliaa) [inline Game/Scripts/PowerUpLogicScript.cpp:65]: Due to de `#ifdef`, this check (and thus the function `GetDebugDraw()`) is not needed.

File: Game/Scripts/PowerUpsManagerScript.h [hunk 2]
```diff
@@ -22,6 +23,9 @@ class PowerUpsManagerScript : public Script
 	const PowerUpType& GetSavedPowerUpType() const;
 	const PowerUpType& GetActivePowerUpType() const;
 
+	const float GetRadiusSeeking();
+	const bool GetDebugDraw();
```
Comment (markmeliaa) [inline Game/Scripts/PowerUpsManagerScript.h:27]: There's no need to add `const` both in this funcion and the function above, they are returning primitives.

**Review by markmeliaa — APPROVED** [review]
Everything works correctly, just a couple comments

**Review by DADSILVER — APPROVED** [review]

File: Game/Scripts/PowerUpLogicScript.cpp (hunk 1 above)
Comment (PaablooCH) [reply Game/Scripts/PowerUpLogicScript.cpp:65]: GetDebugDraw is done to give the ability to the user to disable the debugdraw through the scripts, is just to don't have too much information in the scene

**Pull request merged**